spec/ROOT

```diff
@@ -93,6 +93,7 @@ session CSpec in "cspec" = CKernel +
 
 session CKernel in "cspec/$L4V_ARCH" = CParser +
   sessions
+    "ASpec"
```

Comment:
Does this dependency cost anything time-wise on other arches, or only if they actually pull in theories from ASpec?
There's a concurrency issue with CKernel normally being built in parallel with ASpec, and the xml files will probably try to do that, which could lead to unreliable results.

Reply:
The other arches are unaffected time-wise, because they don't actually draw in any new theories, they only declare the session dependency. You're right with the concurrency issues, though, I hadn't thought about these. If ASpec was not in the path before and it needs regeneration of stuff, there might be interference or CRefine might not see the right version.